metrics-jersey/src/main/java/io/stargate/metrics/jersey/dwconfig/StargateV1ConfigurationSourceProvider.java

```diff
@@ -0,0 +1,66 @@
+package io.stargate.metrics.jersey.dwconfig;
+
+import io.dropwizard.configuration.ConfigurationSourceProvider;
+import io.dropwizard.configuration.FileConfigurationSourceProvider;
+import io.dropwizard.configuration.ResourceConfigurationSourceProvider;
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Objects;
+
+/**
+ * Implementation of {@link ConfigurationSourceProvider} which will try to locate configuration file
+ * using 3 possibilities, in descending order of precedence
+ *
+ * <ol>
+ *   <li>Explicit System Property of form "stargate.configurationFile.${MODULENAME}" specifying
+ *       exact filename of the configuration file to use
+ *   <li>Existence of file "${MODULENAME}-${CONFIGBASE} which is read as the configuration
+ *   <li>Classpath Resource with name "${CONFIGBASE}"
+ * </ol>
+ *
+ * (where "${MODULENAME}" and "${CONFIGBASE} are passed as arguments; former typically being
+ * something like {@code "restapi"} and latter {@code "config.yaml"})
+ */
+public class StargateV1ConfigurationSourceProvider implements ConfigurationSourceProvider {
```

Comment:
and `core` does not work, this is really not the place for this.. but I get why you did it..

Reply:
Agreed, not a great place but best of poor choices. And does seem right to add a new module for 1 class, for Stargate v1 only.